### ISSUE TITLE
The resource is executed each time on CentOS 7

### ISSUE THREAD

**Issue by nicutor** [chef-cookbooks/locale#33]
### Cookbook version
2.0.1

### Chef-client version
13.6.4

### Platform Details
CentOS 7

### Scenario:
-

### Steps to Reproduce:
-

### Expected Result:
Avoid execution everytime chef-client run.

### Actual Result:
locale setup is executed each time chef-client is running because Locale.up_to_date function is not detection the default quotas from CentOS 7:

```
# cat /etc/locale.conf
LANG="en_US.UTF-8"
```

Sugestions to fix:

Replace this line
```
locale.include?("LANG=#{lang}") &&
```
with
```
locale.match?("LANG=([\"']*)#{lang}([\"']*)") &&
```
or
```
(locale.include?("LANG=#{lang}") || locale.include?("LANG=\"#{lang}\"") || locale.include?("LANG='#{lang}'")) &&
```

**Comment by tas50** [comment]
This should be fixed in 2.1. If you're still having this problem please open this back up and I'll take a look

**Comment by nicutor** [comment]
Hi @tas50 ,

The issue is now on CentOS 6. Can you please take a look and fix it?
